### PR TITLE
Сброс клиентского кэша

### PR DESCRIPTION
**Pull request by kinday**

### CODE REVIEW

File: helpers/bust.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { join } from 'path';
+import { fromFileSync as getHash } from 'hasha';
```
Comment (iamstarkov) [inline helpers/bust.js]: why sync?

Comment (kinday) [inline helpers/bust.js]: How should I use async while rendering Jade template, huh?

Comment (iamstarkov) [inline helpers/bust.js]: you are right

File: helpers/bust.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const cwd = process.cwd();
+const cache = {};
```
Comment (iamstarkov) [inline helpers/bust.js]: maybe [`R.memoize`](http://ramdajs.com/0.18.0/docs/#memoize)?

File: helpers/bust.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { join as _join } from 'path';
+import { fromFileSync as _hash } from 'hasha';
+import { memoize } from 'ramda';
+
+const join = _join.bind(null, process.cwd(), 'dist');
+const hash = filepath => _hash(filepath, { algorithm: 'md5' });
+
+export default memoize(url => `${url}?${hash(join(url))}`);
```
Comment (iamstarkov) [inline helpers/bust.js]: what about smth like this?

``` js
import { join } from 'path';
import { fromFileSync as hash } from 'hasha';
import { pipe, memoize, curry, partialRight } from 'ramda';

const curriedJoin = curry(join);

const hash = pipe(
  curriedJoin(process.cwd(), 'dist'),
  partialRight(hash, [{ algorithm: 'md5' }]));

const bust = path => `${path}?${hash(path)}`;

export default memoize(bust);
```

Comment (kinday) [inline helpers/bust.js]: Looks sexy.

**Pull request merged**